Dockerfile

```diff
@@ -1,16 +1,21 @@
-FROM alpine:3.4
+FROM golang:1.11-alpine
```

Comment:
Are we sensitive to alpine versions?

Reply:
Hmm, they have broken compatibility before... I think I would vote for trying to stay on a stable version (you can upgrade it if needed though), unless it's a lot of trouble.